bundles/go/go_completer.moon

```diff
@@ -0,0 +1,50 @@
+import app, config, interact from howl
+import ProcessBuffer from howl.ui
+import Process from howl.io
+import Matcher from howl.util
+append = table.insert
+
+install_gocode = ->
+  p = Process {
+    cmd: { 'go', 'get', '-u', '-v', 'github.com/nsf/gocode' },
+    read_stdout: true,
+    read_stderr: true,
+  }
+
+  buffer = ProcessBuffer p
+  editor = app\add_buffer buffer
+  editor.cursor\eof!
+  buffer\pump!
+  
+run_gocode = (context) ->
+  cmd = { 'gocode', '-f=csv', 'autocomplete', context.pos-1 }
+  status, out, err, process = pcall Process.execute, cmd, stdin: context.buffer.text
+  unless status
+    error out unless out\match 'No such file'
```

Comment:
Instead of attempting to run this and then checking the output, `howl.sys.find_executable` could be used to check for the existence of `gocode`.